libmscore/figuredbass.cpp

```diff
@@ -17,10 +17,13 @@
 #include "system.h"
 #include "segment.h"
 #include "chord.h"
+#include "staff.h"
 #include "rest.h"
 #include "score.h"
 #include "sym.h"
 #include "xml.h"
+#include "utils.h"
+#include "mscore/scoreview.h"
```

Comment:
that’s too bad, becasue the functions in scoreview.h are very handy for auto-score generation.

> On 22 May 2015, at 20:32, Nicolas Froment <notifications@github.com> wrote:
> 
> In libmscore/figuredbass.cpp <https://github.com/musescore/MuseScore/pull/2015#discussion_r30919843>:
> 
> >  #include "rest.h"
> >  #include "score.h"
> >  #include "sym.h"
> >  #include "xml.h"
> > +#include "utils.h"
> > +#include "mscore/scoreview.h"
> This is not allowed. It breaks the fact that libmscore is a library not depending on mscore.
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/musescore/MuseScore/pull/2015/files#r30919843>.
>